tests/unit_tests/training/utils/test_flop_utils.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for flop_utils module."""
+
+import importlib
+import pytest
+import math
+from megatron.bridge.training.utils.flop_utils import num_floating_point_operations
+from megatron.bridge.training.tokenizers.tokenizer import _vocab_size_with_padding
```

Comment:
rebase to use https://github.com/NVIDIA-NeMo/Megatron-Bridge/blob/main/src/megatron/bridge/utils/vocab_utils.py

handling padded vocab sizes needed fixes described in https://github.com/NVIDIA-NeMo/Megatron-Bridge/pull/455